src/preamble.js

```diff
@@ -1022,10 +1022,6 @@ function enlargeMemory() {
   updateGlobalBuffer(replacement);
   updateGlobalBufferViews();
 
-#if ASSERTIONS
```

Comment:
we should still keep it for asm.js, though. please put `#if WASM == 0` before and `#endif // WASM == 0` after, and add a comment inside saying "log js memory growth events because they can be slow".

Reply:
Question: doesn't line 1029 - 1033 already do that? maybe just add ` + ', took ' + (Date.now() - start) + ' ms (has ArrayBuffer.transfer? ' + (!!ArrayBuffer.transfer) + ')'` there?

Reply:
Oh, good point. Yeah, that line is sufficient I think.